baseplate/lib/message_queue.py

```diff
@@ -35,7 +54,40 @@ def __init__(self, inner: Exception):
         super().__init__(f"{inner} (check `ulimit -q`?)")
 
 
-class MessageQueue:
+class QueueType(Enum):
```

Comment:
```suggestion
class QueueType(str, Enum):
```

the string comparison below doesn't work for enums unless they inherit str -- there may be better ways to do this (looks like python got a [StrEnum](https://docs.python.org/3/library/enum.html#enum.StrEnum) type in a later version